api/URL.json

```diff
@@ -28,34 +28,20 @@
           "edge_mobile": {
             "version_added": "12"
           },
-          "firefox": [
-            {
-              "version_added": "19",
-              "notes": [
-                "Firefox had a bug whereby single quotes contained in URLs are escaped when accessed via URL APIs (see <a href='https://bugzil.la/1386683'>bug 1386683</a>). This has been fixed as of Firefox 57.",
-                "To use it from chrome code, JSM and Bootstrap scope, you have to import it with <code>Cu.importGlobalProperties(['URL']);</code>."
-              ]
-            },
-            {
-              "version_added": "4",
-              "version_removed": "18",
-              "notes": "This interface was supported with the non-standard <code>nsIDOMMozURLProperty</code> internal type. However this didn't make any difference in practice, as the only way to access such an object was through <code>window.URL</code>."
-            }
-          ],
-          "firefox_android": [
-            {
-              "version_added": "19",
-              "notes": [
-                "Firefox had a bug whereby single quotes contained in URLs are escaped when accessed via URL APIs (see <a href='https://bugzil.la/1386683'>bug 1386683</a>). This has been fixed as of Firefox 57.",
-                "To use it from chrome code, JSM and Bootstrap scope, you have to import it with <code>Cu.importGlobalProperties(['URL']);</code>."
-              ]
-            },
-            {
-              "version_added": "14",
-              "version_removed": "18",
-              "notes": "This interface was supported with the non-standard <code>nsIDOMMozURLProperty</code> internal type. However this didn't make any difference in practice, as the only way to access such an object was through <code>window.URL</code>."
-            }
-          ],
+          "firefox": {
+            "version_added": "19",
+            "notes": [
+              "Firefox had a bug whereby single quotes contained in URLs are escaped when accessed via URL APIs (see <a href='https://bugzil.la/1386683'>bug 1386683</a>). This has been fixed as of Firefox 57.",
```

Comment:
I realize you didn't write this, but since we're cleaning things up anyway: can these notes start with the version info? For example, `Before Firefox 57, Firefox had a bug where…`. After all, the bug will become increasingly irrelevant as time goes on